code/game/objects/structures/displaycase.dm

```diff
@@ -28,17 +54,34 @@
 				src.healthcheck()
 
 
+/obj/structure/displaycase/proc/open_close(mob/user)
+	open = !open
+	alarm = !alarm
+	visible_message("<span class='notice'>The display case is now [src.open ? "opened" : "closed"].</span>")
+	update_icon()
+
 /obj/structure/displaycase/bullet_act(var/obj/item/projectile/Proj)
 	health -= Proj.get_structure_damage()
 	..()
 	src.healthcheck()
 	return
 
+/obj/structure/displaycase/proc/tripalarm(security_level)
+	var/area/case_area = get_area(src)
+	switch(security_level)
+		if(1)
+			visible_message("<span class='notice'>[src] pings cheerfully.</span>", "<span class='notice'>You hear a ping.</span>")
+		if(2)
+			priority_announcement.Announce("Warning, a company asset has been removed from it's assigned storage case at [case_area.name]. Company assets require command approval to remove from protected cases.", new_sound = 'sound/ai/caseseclevel1.ogg')
+		if(3)
+			set_security_level(SEC_LEVEL_BLUE)
+			priority_announcement.Announce("Warning, a company asset has been removed from the storage case in the [case_area.name]. A blue alert level has been enacted to allow security to recover the asset. All crew are advised to not impede security's inquiry into the assets whereabouts.", new_sound = 'sound/ai/caseseclevel2.ogg')

```

Comment:
This does not look the same as the like above.

`Warning, a company asset has been removed from it's assigned storage case at [case_area.name]`

and 

`Warning, a company asset has been removed from the storage case in the [case_area.name]`

I would say pick either